edx_sga/management/commands/sga_migrate_submissions.py

```diff
@@ -1,3 +1,6 @@
+"""
+SGA migration submissions file
```

Comment:
"Django command which migrates existing SGA submissions for a course from old SGA implementation to newer version that uses the 'submissions' application."